### PR TITLE
:seedling: Supports AD-mastered user password expiry warning

### PR DESCRIPTION
**Pull request by gowthamidommety-okta**
image001

### CODE REVIEW

**Comment by gowthamidommety-okta** [comment]
@mauriciocastillosilva-okta @mpeng-okta Please review

File: packages/@okta/i18n/package.json [hunk 1]
```diff
@@ -1,30 +1,31 @@
...
+      "@okta/i18n@34.5.0",
       "/home/vagrant/okta/okta-signin-widget"
     ]
   ],
-  "_from": "@okta/i18n@34.3.0",
-  "_id": "@okta/i18n@34.3.0",
+  "_from": "@okta/i18n@34.5.0",
+  "_id": "@okta/i18n@34.5.0",
   "_inCache": true,
+  "_installable": true,
```
Comment (mauriciocastillosilva-okta) [inline packages/@okta/i18n/package.json]: this should probably not be here, can you remove it?

File: packages/@okta/i18n/package.json [hunk 2]
```diff
@@ -1,30 +1,31 @@
...
+    "raw": "@okta/i18n@34.5.0",
+    "rawSpec": "34.5.0",
     "scope": "@okta",
-    "spec": "34.3.0",
+    "spec": "34.5.0",
     "type": "version"
   },
   "_requiredBy": [
-    "/"
+    "#USER"
```
Comment (mauriciocastillosilva-okta) [inline packages/@okta/i18n/package.json]: I have seen this in a few PRs, but if I remember correctly this is not needed, it should be ```/```

File: src/CustomPasswordExpiredController.js [hunk 3]
```diff
@@ -35,6 +35,9 @@ function (Okta, FormController, FormType, Footer, Util) {
         else if (expiringSoon && numDays === 0) {
           return Okta.loc('password.expiring.today', 'login');
         }
+        else if (expiringSoon && (numDays === null || numDays === undefined)) {
```
Comment (mauriciocastillosilva-okta) [inline src/CustomPasswordExpiredController.js]: Do we need to check if ```numDays``` is null or underfined? Wouldn't it be enough to just check if ```expiringSoon``` is true?  (same comment for the other controller)
```
else if (expiringSoon) {
```

Comment (mpeng-okta) [reply src/CustomPasswordExpiredController.js]: Good point, we only need to check `expiringSoon`, I believe. I checked both expired and non-expired cases to make sure.

Comment (gowthamidommety-okta) [reply src/CustomPasswordExpiredController.js]: 👍 Thanks Michael and Mau. I'll change that

**Pull request merged**